package/base-files/files/lib/upgrade/do_stage2

```diff
@@ -5,13 +5,13 @@
 include /lib/upgrade
 
 v "Performing system upgrade..."
-if type 'platform_do_upgrade' >/dev/null 2>/dev/null; then
+if type 'platform_do_upgrade' > /dev/null 2> /dev/null; then
```

Comment:
2>&1 /dev/null

Reply:
This should be replaced my `2>&1 > /dev/null` right?

Reply:
Yeah I think so.